src/main/java/net/dries007/tfc/objects/items/ItemsTFC.java

```diff
@@ -220,7 +220,7 @@ public static void registerItems(RegistryEvent.Register<Item> event)
 
         simpleItems.add(register(r, "crop/product/jute", new ItemMisc(Size.TINY, Weight.LIGHT), CT_MISC));
         simpleItems.add(register(r, "crop/product/jute_fiber", new ItemMisc(Size.TINY, Weight.LIGHT), CT_MISC));
-        simpleItems.add(register(r, "crop/product/burlap_cloth", new ItemMisc(Size.TINY, Weight.LIGHT, "cloth"), CT_MISC));
+        simpleItems.add(register(r, "crop/product/burlap_cloth", new ItemMisc(Size.TINY, Weight.LIGHT, "cloth", "cloth_low_quality"), CT_MISC));
```

Comment:
Does TFC use these ore dict names? If not, then remove them.

Reply:
Yeah, some of them are unnecessary, will be removed